fabbwled-backend/src/main/resources/sections/433-468.yaml

```diff
@@ -0,0 +1,132 @@
+- number: 440
+  content:
+    - text: |
+        Becoming an initiate of Elnir gives you the benefit of paying less for blessings and other services the Temple can offer.
+        It costs 60Shards to become and initiate. You cannot become an initiate of Elnir if you are already an initiate of another temple.
+        If you choose to become an initiate, write Elnir in the Godbox on your Adventure Sheet – and then cross 60 shards off your Adventure Sheet.
+    - turnTo: 568
+- number: 441
+  content:
+    - text: |
+        You clamber up on to the battlements of the Temple of Tyrnai with relative ease and manoeuvre yourself
+        into a position overlooking the golem guards at the gate. Now you have to reach down without being noticed.
+    # TODO: Add dice functionality
+- number: 442
+  content:
+    - text: |
+        You set out to explore the city of Yellowport at night,unwholesome though it is with its reeking air and dusty ochrestreets.
+    - choice:
+        - text: Head for the rough part of the town
+          turnTo: 21
+        - text: Go to the wealthy area
+          turnTo: 178
+        - text: Explore the merchants’ storehouses
+          turnTo: 265
+- number: 443
+  content:
+    - text: |
+        You are witness to a major sea battle, involving at least 40warships, between the Sokaran navy,
+        and a pirate fleet. Arrows fill the air,
+        and smoking fireballs are launched from catapults mounted on some of the ships.
+        The shrieks of the dying carry across the waves.  
+        You decide it would be better not to get involved in such a conflict
+    - turnTo: 420
+- number: 444
+  content:
+    - text: |
+        You try to make a run for it. 
+        Roll two dice and add your Rank. Subtract 1 from the total if you have a poor crew. 
+        Add 1 if you have a good crew or add 2 if you have an excellent crew.
+    # TODO: Add dice functionality
+- number: 445
+  content:
+    - text: |
+        You climb up, and squeeze through an open window into along hall.
+        Thin wires are stretched across the room, 
+        but you are an able enough rogue to crawl through without touching any of them and setting off possible traps or alarms.
+        Inside the temple, it is cool and dark, filled with an unearthly stillness.
+        Suddenly, the temple doors are flung open, and two figures lumber into the room.
+        You realize with a thrill of horror that the bull-headed iron statues outside the gates have come to life. 
+        They are golems, set here to guard the temple.
+    - choice:
+        - text: Run for your life
+          turnTo: 349
+        - text: Fight them
+          turnTo: 569
+- number: 446
+  content:
+    - if:
+        isTickBoxDone: true
+      then:
+        - turnTo: 226
+      else:
+        - text: |
+            You set out that night, on your quest. Make a MAGIC roll atDifficulty 10.
+        - checkTickBox: true
+        # TODO: Add dice functionality
```

Comment:
also no termination

Reply:
yeah not easy if the schema validation doesn't help you